bin/cml-tensorboard-dev.js

```diff
@@ -68,7 +68,7 @@ const run = async (opts) => {
     const regex = /(https?:\/\/[^\s]+)/;
     const matches = stdoutData.match(regex);
 
-    if (matches.length) {
+    if (matches && matches.length) {
```

Comment:
This could definitely solve the error... but the loop? I think that there is a retry in a loop to endup with that result

Reply:
Indeed the error has to be that the output is not what is expected

Reply:
The loop will break after 60 seconds of invalid results. I think that it's the expected behavior?

Reply:
This was broken because `"hello".match(/(world)/) === null` and it errored each time it didn't find a valid link.

Reply:
The question is... why it should not receive the link?

Reply:
Because of some error we're missing due to the lack of debug output? If you're able to reproduce this issue consistently, it would be nice to take a look at the full standard output/error.